dialer.go

```diff
@@ -195,17 +209,20 @@ func (d *Dialer) Dial(ctx context.Context, instance string, opts ...DialOption)
 		endInfo(err)
 		return nil, err
 	}
-	addr, tlsCfg, err := i.ConnectInfo(ctx, cfg.ipType)
+	ci, err := i.ConnectInfo(ctx, cfg.ipType)
```

Comment:
We already have [`i.InstanceVersion`](https://github.com/GoogleCloudPlatform/cloud-sql-go-connector/blob/97e4b6538bf085148c90016cce21876e03be97e5/internal/cloudsql/instance.go#L223) - should we just that instead of mutating this return?

Reply:
Woops -- missed that.